crates/solidity/inputs/language/src/definition.rs

```diff
@@ -503,7 +503,7 @@ codegen_language_macros::compile!(Language(
                             definitions = [KeywordDefinition(
                                 value = Sequence([
                                     Atom("bytes"),
-                                    Choice([
+                                    Optional(Choice([
```

Comment:
Does this also apply to `YulBytesKeyword`?

Reply:
Ah, I knew I forgot about something. It does! Let me fix this.

Reply:
Done in 8e0a6181ac08894796bc070e44e0a32c226bd1b9